src/core/algorithms/gfd/egfd_validation.cpp

```diff
@@ -1021,8 +1025,9 @@ bool Validate(graph_t const& graph, Gfd const& gfd) {
     BottomUpRefinement(cpi, graph, pat, levels, parent, candidates);
     auto elapsed_milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(
             std::chrono::system_clock::now() - start_time);
-    std::cout << "CPI constructed in " << elapsed_milliseconds.count() << ". Matching..."
-              << std::endl;
+
+    // std::cout << "CPI constructed in " << elapsed_milliseconds.count() << ". Matching..."
+    //           << std::endl;

```

Comment:
use `LOG(DEBUG)` here and on other places
or you can remove some logging functions if you are sure that they are no longer needed